tests/accuracy_utils.py

```diff
@@ -127,6 +127,15 @@ def to_reference(inp, upcast=False):
     return ref_inp
 
 
+def to_reference_gpu(inp, upcast=False, device='musa'):
```

Comment:
is it used in test code?

Reply:
confirmed, should be removed, we will fix it.